### DISCUSSION TITLE
Connecting to another server using different windows credentials

### DISCUSSION THREAD

**Discussion by agentwalker** [microsoft/azuredatastudio#23470] [Q&A]
In Azure data studio I'm trying to connect to a number of SQL server instances however these are all setup for windows authentication only. Also the windows account setup on these instances is not the one I'm currently logged in as.

So
My local SQL server instance
Windows Authentication
MyDomain\John.Doe - this is what I'm currently logged into windows as.

External SQL Instance
Windows Authentication
AnotherDomain\Mike.Jame

At the moment the only way I can get round this is to start Azure Data Studio with the separate windows authentication depending on which SQL instance I need.

Is there a way to configure this within Data studio via the connections options (Can't see any at the moment)

Thanks for any help

**Comment by agentwalker** [comment]
Another workaround ive just discovered is to use Credential Manager and add entries for the server instances with the port that SQL servers running as. It seems to work quite well.

**Comment by erinstellato-ms** [comment]
@agentwalker You are correct in that there is no way within the connection dialog to specify a different Windows domain and username.  We do have a request for this functionality (https://github.com/microsoft/azuredatastudio/issues/112), please upvote it and add your use case as this helps us prioritize issues (and we are actively making connectivity improvements).  Thanks.
